pyccel/errors/messages.py

```diff
@@ -111,6 +111,7 @@
 PYCCEL_RESTRICTION_LIST_COMPREHENSION_SIZE = 'Could not deduce the size of this list comprehension. If you believe this expression is simple then please create an issue at https://github.com/pyccel/pyccel/issues and provide a small example of your problem.'
 PYCCEL_RESTRICTION_LIST_COMPREHENSION_LIMITS = 'Pyccel cannot handle this list comprehension. This is because there are occasions where the upper bound is smaller than the lower bound for variable {}'
 PYCCEL_RESTRICTION_INHOMOG_LIST = 'Inhomogeneous lists are not supported by Pyccel. Please use a tuple'
+PYCCEL_RESTRICTION_INHOMOG_SET = 'Inhomogeneous Sets are not supported by Pyccel. Please use a tuple'
```

Comment:
How can you use a tuple in place of a set?

Reply:
I am not convinced that tuples and sets are sufficiently similar for tuples to be a drop-in replacement. For example I often use sets to remove duplicate items in a list. How can I do this with a tuple?

Reply:
I get it now I'm going to change it